style.css

```diff
@@ -0,0 +1,382 @@
+  margin: 3px;
+  width: 30px;
+  border: 1px solid gray;
+}
+
+.see-more {
+  border: 1px solid gray;
+  margin: 15px;
+  color: gray;
+}
+
+.see-more > p {
+  margin-bottom: 10px;
+}
+
+.see-more > p > i {
+  color: rgb(255, 64, 0);
+  padding-left: 4px;
+}
+
+.under {
+  text-decoration: underline;
+}
+
+.summit-sub {
+  background-color: rgba(255, 64, 0, 0.573);
+  height: 40vh;
+  padding-top: 20%;
+}
+
+.summit-one {
+  background-image: url("assets/pastImg1.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 40vh;
+}
+
+.summit-two {
+  background-image: url("assets/pastImg2.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 40vh;
+}
+
+.more-partners {
+  background-image: url("assets/Headphones.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.first-group {
+  display: flex;
+  justify-content: space-around;
+}
+
+.first-p,
+.second-p,
+.third-p,
+.fourth-p,
+.fifth-p {
+  font-size: 30px;
+  font-weight: 700;
+  color: aquamarine;
+}
+
+.first-p {
+  color: rgba(255, 64, 0, 0.849);
+}
+
+.third-p {
+  color: rgba(255, 64, 0, 0.854);
+}
+
+.bottom-part {
+  display: flex;
+}
+
+.mobile-nav {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 100vh;
+  transition: 0.2s ease-in-out;
+  background-color: rgba(255, 64, 0, 0.8);
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
+  display: none;
+}
+
+.mobile-nav-ul > li > a {
+  color: #fff;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  text-decoration: none;
+}
+
+.nav-toggle {
+  display: flex;
+  flex-direction: column;
+}
+
+.mobile-nav-ul > li {
+  list-style-type: none;
+}
+
+.cross {
+  color: #fff;
+  font-size: 34px;
+  padding: 0 7px;
+  float: right;
+  margin-left: 85%;
+}
+
+.summit-sub:hover {
+  opacity: 0;
+  cursor: pointer;
+}
+
+@media only screen and (max-width: 768px) {
+  #sponsor,
+  #updates {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .main-nav {
+    display: flex;
+  }
+
+  .main-nav-ul {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 0;
+    float: right;
+  }
+
+  .main-nav-i {
+    color: #fff;
+    list-style: none;
+    margin-left: 20px;
+  }
+
+  .main-nav-i:hover {
+    color: aquamarine;
+    list-style: none;
+    margin-left: 20px;
+    cursor: pointer;
+  }
+
+  #menu-image,
+  .see-more,
+  .greeting-about,
+  .more-partners-about {
+    display: none;
+  }
+
+  .navigation {
+    display: contents;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .second-nav-ul {
+    display: flex;
+    margin-top: 10px;
+    justify-content: flex-end;
+  }
+
+  .second-nav-ul > li {
+    list-style: none;
+    padding-left: 30px;
+  }
+
+  .second-nav-i {
+    color: black;
+    text-decoration: none;
+  }
+
+  .second-nav-i:hover {
+    color: rgb(255, 64, 0);
```

Comment:
- [ ] As suggested already, I think you should use the color in the Behance design, the color used in the design is `#EC5242`  instead of `rgb(255, 64, 0)`:pray: